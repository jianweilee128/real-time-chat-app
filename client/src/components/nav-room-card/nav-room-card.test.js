import React from "react";
import { shallow } from "enzyme";

import NavRoomCard from "./nav-room-card.component";

describe("NavRoomCard component", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<NavRoomCard.WrappedComponent />);
  });

  it("should render NavRoomCard component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
