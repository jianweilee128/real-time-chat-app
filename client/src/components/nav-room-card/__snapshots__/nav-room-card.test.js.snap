// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavRoomCard component should render NavRoomCard component 1`] = `ShallowWrapper {}`;
